packages/joy-actors/src/ActorsList/index.tsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react'
+import { BareProps } from '@polkadot/ui-app/types';
+import { ComponentProps } from '../props';
+import { withCalls } from '@polkadot/ui-api/index';
+import { Table } from 'semantic-ui-react';
+import { Option} from '@polkadot/types';
+import AddressMini from '@polkadot/ui-app/AddressMiniJoy';
+
+type Props = BareProps & ComponentProps;
+
+export default class ActorsList extends React.PureComponent<Props> {
+    render() {
+        const actors: Array<string> = this.props.actors;
+
+        return (
+            <Table>
+                <Table.Header>
+                    <Table.Row>
+                        <Table.HeaderCell>Role</Table.HeaderCell>
+                        <Table.HeaderCell>Actor Account</Table.HeaderCell>
+                        <Table.HeaderCell>MemberId</Table.HeaderCell>
```

Comment:
MemberId -> Member Id
```suggestion
                        <Table.HeaderCell>Member Id</Table.HeaderCell>
```